From: Calculinux <noreply@calculinux>
Subject: dt: drop default CMA reservation on PicoCalc

Stock PicoCalc SPI DBI panel does not need CMA. DSI overlays set &cma.

---

--- a/linux-rk3506-luckfox-lyra.dtsi
+++ b/linux-rk3506-luckfox-lyra.dtsi
@@ -102,7 +102,7 @@
 
 /**********memory**********/
 &cma {
-	size = <0x2000000>;
+	size = <0x0>;
 };
 
 /**********graphics**********/
--- a/linux-rk3506g-luckfox-lyra.dts
+++ b/linux-rk3506g-luckfox-lyra.dts
@@ -25,7 +25,7 @@
 
 /**********display**********/
 &cma {
-	size = <0x1000000>;
+	size = <0x0>;
 };
 
 &dsi {
